### ISSUE TITLE
Unable to configure handler 'sentry' with raven-python 6.1.0 and django 1.5

### ISSUE THREAD

**Issue by JeanFred** [getsentry/raven-python#1013]
I have tried upgrading the raven client from 6.0.0 to 6.1.0 on my Django 1.5 app (yes, it’s old :) and hit the following error (when running it with UWSGI):
```
Error in sys.excepthook:
Traceback (most recent call last):
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/raven/base.py", line 276, in handle_exception
    self.captureException(exc_info=exc_info, level='fatal')
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/raven/base.py", line 818, in captureException
    'raven.events.Exception', exc_info=exc_info, **kwargs)
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/raven/contrib/django/client.py", line 292, in capture
    result = super(DjangoClient, self).capture(event_type, **kwargs)
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/raven/base.py", line 638, in capture
    **kwargs)
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/raven/contrib/django/client.py", line 233, in build_msg
    data = super(DjangoClient, self).build_msg(*args, **kwargs)
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/raven/base.py", line 451, in build_msg
    data['modules'] = self.get_module_versions()
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/raven/base.py", line 303, in get_module_versions
    modules = get_versions(self.include_paths)
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/raven/utils/__init__.py", line 112, in get_versions
    __import__(module_name)
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/raven/contrib/django/__init__.py", line 19, in <module>
    initialize()
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/raven/contrib/django/models.py", line 257, in initialize
    get_client()  # NOQA
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/raven/contrib/django/models.py", line 133, in get_client
    instance = Client(**options)
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/raven/contrib/django/client.py", line 140, in __init__
    self.install_sql_hook()
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/raven/contrib/django/client.py", line 143, in install_sql_hook
    install_sql_hook()
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/raven/utils/__init__.py", line 185, in new_func
    rv = func(*args, **kwargs)
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/raven/contrib/django/client.py", line 81, in install_sql_hook
    from django.db.backends.utils import CursorWrapper
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/django/db/__init__.py", line 40, in <module>
    backend = load_backend(connection.settings_dict['ENGINE'])
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/django/db/__init__.py", line 34, in __getattr__
    return getattr(connections[DEFAULT_DB_ALIAS], item)
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/django/db/utils.py", line 93, in __getitem__
    backend = load_backend(db['ENGINE'])
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/django/db/utils.py", line 45, in load_backend
    raise ImproperlyConfigured(error_msg)
django.core.exceptions.ImproperlyConfigured: 'django.db.backends.dummy' isn't an available database backend.
Try using 'django.db.backends.XXX', where XXX is one of:
    u'mysql', u'oracle', u'postgresql_psycopg2', u'sqlite3'
Error was: cannot import name util

Original exception was:
Traceback (most recent call last):
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/myapp/wsgi.py", line 23, in <module>
    import django.test
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/django/test/__init__.py", line 5, in <module>
    from django.test.client import Client, RequestFactory
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/django/test/client.py", line 21, in <module>
    from django.db import close_connection
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/django/db/__init__.py", line 11, in <module>
    if settings.DATABASES and DEFAULT_DB_ALIAS not in settings.DATABASES:
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/django/conf/__init__.py", line 52, in __getattr__
    self._setup(name)
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/django/conf/__init__.py", line 48, in _setup
    self._configure_logging()
  File "/usr/lib/myapp/local/lib/python2.7/site-packages/django/conf/__init__.py", line 82, in _configure_logging
    logging_config_func(self.LOGGING)
  File "/usr/lib/python2.7/logging/config.py", line 794, in dictConfig
    dictConfigClass(config).configure()
  File "/usr/lib/python2.7/logging/config.py", line 576, in configure
    '%r: %s' % (name, e))
ValueError: Unable to configure handler 'sentry': Cannot resolve 'raven.contrib.django.handlers.SentryHandler': cannot import name util
```

Downgrading to 6.0.0 solved the issue.

**Comment by tomkins** [comment]
This was fixed in #1033.

**Comment by tomkins** [comment]
Actually, #1033 does stop this issue - however it just masks the real problem. Exceptions from `manage.py` commands don't get reported.

The `ImportError` occurs at: https://github.com/getsentry/raven-python/blob/master/raven/contrib/django/serializers.py#L63, but gets caught by https://github.com/getsentry/raven-python/blob/master/raven/contrib/django/models.py#L262

I'll see if I can diagnose it further and raise a new issue for it.
